### DISCUSSION TITLE
Passport details

### DISCUSSION THREAD

**Discussion by joeedhorton** [UKHomeOffice/design-system#365] [Ideas]
## What
Ask users for passport details

## Why
Currently, there are a few variations being used in Immigration and elsewhere. The goal should be to reduce the amount of variations and establish a consistent way of asking for passport details, whilst taking into account the potential differences in the pattern depending on the context and policy of the service. 

**The patterns below have been taken from the ITP Design System Trello board**

**Comment by joeedhorton** [comment]
### Version 1

![passport-details](https://user-images.githubusercontent.com/55453490/86343538-00cc8e00-bc51-11ea-8a1c-b243104b2ae2.png)
> From EUSS, Apply

#### Research and findings
- System feedback shows that a common error for users is submitting their passport number. 
- The hint text was introduced to show users what to expect when looking for their passport number. 
- The number of error messages shown on this page decreased after this hint text was introduced.

#### Why this pattern
- Tested with a range of users from different countries (both EU and non-EU).
- Content can be question format if the expiry date isn't needed (see variation below)

More research needed into how to ensure users input the correct number. 
**Currently users have to enter the number again on a separate screen.**

**Comment by joeedhorton** [comment]
### Version 2

![passport-details-v2](https://user-images.githubusercontent.com/55453490/86346546-fe6c3300-bc54-11ea-99cb-8b3585b32dad.png)
> From EUSS, Check Status

#### How this is different to version 1 (above)
- Expiry date is not asked
- H1 is a question because expiry date isn't needed and it should be consistent with the GDS Design System

#### Questions
- Which hint text has shown to have better results?
> Your passport number may include letters
OR
> Can contain letters for example, 120382978A

**Comment by joeedhorton** [comment]
### Version 3
This variation of the proposed pattern is content only and was an output from a design crit session. I believe this has come out of AUK1. **This version has not been used in a service and has not had an input from an interaction design.** 

```
H1: Your passport/Your partner's passport details

h2: Name on the passport
Use the English spelling if it is written in 2 languages.

Given name
Hint text: Your/Their first and middle names

Surname or family name
Hint text: If you/they have more than one surname, enter them all

Heading: Date of birth on the passport
For example, 31 3 1980
Day Month Year

Heading: Passport number
Hint text: May include letters. For example, XXXXXX

Heading: Country of issue
Textbox: autofill with GOV.UK country register list

Heading: Issue date
For example, 30 3 2016
Three text boxes

Heading: Does the passport have an expiry date?
Radio: Yes
Field text: Passport expiry date
Hint text: For example, 31 3 2024
Three text boxes
Radio: No
```

### Questions 
- When and why would this version be used over version 1 or 2?
- When a user inputs their passport number, do we always give them a default example in the hint text (_such as 120382978A_) or does it change depending on their passport/id number?
- Has anyone worked on a service where they've implemented a version such as the one above?

**Comment by catrionabudge** [comment]
A couple more variations from account creation and visa application forms
<img width="438" alt="Screenshot 2021-03-05 at 16 26 40" src="https://user-images.githubusercontent.com/53616961/110148845-3d9ae700-7dd5-11eb-8f80-dfbf73e58a94.png">
<img width="438" alt="Screenshot 2021-03-05 at 16 27 15" src="https://user-images.githubusercontent.com/53616961/110148853-3ecc1400-7dd5-11eb-8a8d-da52800d7b65.png">

**Comment by tomyems** [comment]
Replaced by: https://github.com/UKHomeOffice/design-system/discussions/449
